cico_do_docker_tag.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
This file should be renamed in `cico_do_docker_build_and_push.sh` as these are the more important operations.